app/javascript/src/components/Dashboard/Notes/Form/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui";
+import { Input, Textarea, Select } from "neetoui/formik";
+import { useTranslation } from "react-i18next";
+
+import notesApi from "apis/notes";
+
+import { INITIAL_FORM_VALUES, TAGS, VALIDATION_SCHEMA } from "./constants";
+
+const Form = ({ onClose, refetch }) => {
+  const { t } = useTranslation();
+
+  const handleSubmit = async values => {
+    try {
+      await notesApi.create(values);
+      refetch();
+    } catch (err) {
+      logger.error(err);
+    } finally {
+      onClose();
+    }
+  };
+
+  return (
+    <Formik
+      initialValues={INITIAL_FORM_VALUES}
+      validationSchema={VALIDATION_SCHEMA}
+      onSubmit={handleSubmit}
+    >
+      {({ isSubmitting }) => (
+        <FormikForm noValidate className="w-full">
```

Comment:
Why do we need the `noValidate` class here?

Reply:
This was done to disable [validation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes) from the HTML `form` element.

With `noValidate`            |  Without `noValidate`
:-------------------------:|:-------------------------:
<img width="400" alt="Screenshot 2023-05-29 at 2 15 03 PM" src="https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/assets/48003391/4c7ac161-3c2e-4e58-82e1-d6f88e9dca86">  |  <img width="400" alt="Screenshot 2023-05-29 at 2 19 14 PM" src="https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/assets/48003391/00651e50-782c-451c-a5d2-d04fd8fea581">

Reply:
Got it. Thanks for clarifying.